### PR TITLE
[HB-6851] Tapjoy SDK 13.3.0

### PR DESCRIPTION
**Pull request by cb-jpadilla**
- Updated Tapjoy SDK version to 13.3.0
- Updated Changelog, README, build script to 4.13.3.0.0

### CODE REVIEW

File: TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt [hunk 1]
```diff
@@ -162,30 +170,39 @@ class TapjoyAdapter : PartnerAdapter {
...
+                PartnerLogController.log(GDPR_UNKNOWN)
+                TJStatus.UNKNOWN
+            }
+        }
 
-        Tapjoy.getPrivacyPolicy()?.setUserConsent(
-            when (gdprConsentStatus) {
-                GdprConsentStatus.GDPR_CONSENT_GRANTED -> "1"
-                else -> "0"
+        Tapjoy.getPrivacyPolicy()?.userConsent = when (gdprConsentStatus) {
```
Comment (cb-jpadilla) [inline TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt:190]: Tapjoy now uses Enum Constants to be passed to their privacy setters.

https://dev.tapjoy.com/en/android-sdk/User-Privacy#id-4-implementation

File: TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt [hunk 2]
```diff
@@ -205,7 +222,7 @@ class TapjoyAdapter : PartnerAdapter {
             else CCPA_CONSENT_DENIED
         )
 
-        Tapjoy.getPrivacyPolicy().setUSPrivacy(privacyString)
+        Tapjoy.getPrivacyPolicy().usPrivacy = privacyString
```
Comment (bichenwang) [inline TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt:225]: why is it `Tapjoy.getPrivacyPolicy()?` some of the time and `Tapjoy.getPrivacyPolicy()` some other time?

File: TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt [hunk 3]
```diff
@@ -162,30 +170,39 @@ class TapjoyAdapter : PartnerAdapter {
...
+                TJStatus.UNKNOWN
             }
-        )
+
+            GdprConsentStatus.GDPR_CONSENT_GRANTED -> {
+                PartnerLogController.log(GDPR_CONSENT_GRANTED)
+                TJStatus.TRUE
+            }
+
+            GdprConsentStatus.GDPR_CONSENT_DENIED -> {
```
Comment (bichenwang) [inline TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt:201]: this is super minor and not really a concern, but your order for subject to gdpr and user consent is in a different order xD
though what might be a good practice is always using an `else` clause in a `when` statement.

**Review by bichenwang — APPROVED** [review]
lgtm

File: TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt (hunk 3 above)
Comment (cb-jpadilla) [reply TapjoyAdapter/src/main/java/com/chartboost/mediation/tapjoyadapter/TapjoyAdapter.kt:201]: So, basically remove the `GdprConsentStatus.GDPR_CONSEN_UNKNOWN` case  and have the `else` catch the rest?

**Pull request merged**